runtimes/eoapi/raster/eoapi/raster/app.py

```diff
@@ -95,6 +99,10 @@ async def lifespan(app: FastAPI):
     docs_url="/api.html",
     root_path=settings.root_path,
     lifespan=lifespan,
+    swagger_ui_init_oauth={
+        "clientId": auth_settings.client_id,
```

Comment:
when client_id is set to `""`  does Swagger understand that its unavailable?

Reply:
The `clientId` property populates the value on the auth form's text input ([docs](https://swagger.io/docs/open-source-tools/swagger-ui/usage/oauth2/)). As such, an empty string is equivalent to `null`